### PR TITLE
[Internal] Pipelines: Fixes Multi-Region CI pipeline to include env variable

### PR DESCRIPTION
**Pull request by NaluTripician**
# Pull Request Template

## Description

Adds inclusion of multi-region account connection string to pipeline environment variables

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTests.cs [hunk 1]
```diff
@@ -924,6 +924,19 @@ public void PooledConnectionLifetimeTest()
...
             Assert.AreEqual(socketHandlerType, clientMessageHandlerType);
         }
+
+        [TestMethod]
+        [TestCategory("MultiRegion")]
+        public async Task MultiRegionAccountTest()
+        {
+            string connectionString = TestCommon.GetMultiRegionConnectionString();
+            Assert.IsNotNull(connectionString);
+            CosmosClient cosmosClient = new CosmosClient(connectionString);
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTests.cs:935]:
```suggestion
            using CosmosClient cosmosClient = new CosmosClient(connectionString);
```

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTests.cs [hunk 2]
```diff
@@ -924,6 +924,19 @@ public void PooledConnectionLifetimeTest()
...
+            Assert.IsNotNull(cosmosClient);
+            AccountProperties properties = await cosmosClient.ReadAccountAsync();
+            Assert.IsNotNull(properties);
+            cosmosClient.Dispose();
```
Comment (ealsur) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/ClientTests.cs:939]:
```suggestion
```

**Review by ealsur — APPROVED** [review]

**Review by ealsur — DISMISSED** [review]

**Review by ealsur — APPROVED** [review]

File: Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/Utils/TestCommon.cs [hunk 3]
```diff
@@ -73,6 +73,11 @@ internal static (string endpoint, string authKey) GetAccountInfo()
             return (endpoint, authKey);
         }
 
+        internal static string GetMultiRegionConnectionString()
+        {
+            return Environment.GetEnvironmentVariable("COSMOSDB_MULTI_REGION");
```
Comment (kundadebdatta) [inline Microsoft.Azure.Cosmos/tests/Microsoft.Azure.Cosmos.EmulatorTests/Utils/TestCommon.cs:78]: Can we use the `ConfigurationManager` and have a default value for the environment variable ?

image

**Review by kundadebdatta — DISMISSED** [review]
LGTM!

**Review by kundadebdatta — APPROVED** [review]

**Review by ealsur — APPROVED** [review]

**Pull request merged**